packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.Props.ts

```diff
@@ -59,4 +59,27 @@ export interface IBaseAutoFillProps extends React.HTMLAttributes<HTMLInputElemen
    */
   enableAutoFillOnKeyPress?: KeyCodes[];
 
+  /**
+   * the default value to be visible
+   */
+  defaultVisibleValue?: string;
+
+  /**
+   * Handler for checking and updating the value if needed
+   *  in componentWillReceiveProps
+   *
+   * @param {IBaseAutoFillProps} defaultVisibleValue - the defaultVisibleValue that got passed
+   *  in to the auto fill's componentWillReceiveProps
+   * @returns {string} - the updated value to set, if needed
+   */
+  updateValueInWillReceiceProps?: () => string;
```

Comment:
Fixed the typo. The name is being explicit about where the callback is being plugged in to. This functionality was created so that I could reuse the class instead of having to have almost duplicate functionality except for the changes in these lifecycle functions